doc/source/installation/post-installation.rst

```diff
@@ -0,0 +1,177 @@
+
+    $ mysql_secure_installation
+
+Checking the status and Accessing the Server
+---------------------------------------------
+
+After installation, the server starts. Use the following command to check the status:
+
+.. code-block:: bash
+
+    $ sudo service mysql status
+    
+Access the server with the following command:
+
+.. code-block:: bash
+
+    $ mysql -u root -p
+    
+Configuring the Server to Start at Startup
+------------------------------------------------------
+
+You can manage the server with systemd. If you have installed the server from a generic binary distribution on an operating system that uses systemd, you can manually configure systemd support. 
+
+The following commands start, check the status, and stop the server:
+
+.. code-block:: bash
+
+   $ systemctl start mysql
+   $ systemctl status 'mysql'
+   $ systemctl stop mysql
+
+Enabling the server to start at startup, run the following:
+
+.. code-block:: bash
+
+   systemctl enable mysql
+
+Testing the Server
+------------------
+
+After you have initialized the data directory, and the server is started, you can run a few tests to make sure the server works. 
+
+This section assumes you have used the default installation settings. If you have modified the installation, you should navigate to the installation location. You can also add the location by `Setting the Environment Variables <https://dev.mysql.com/doc/refman/5.7/en/setting-environment-variables.html>`_.
+
+You can use the `mysqladmin <https://dev.mysql.com/doc/refman/5.7/en/mysqladmin.html>`_ client to access the server. 
+
+If you are unable to connect to the server, you should specify the ``root`` user and the root user's password.
+
+.. code-block:: bash
+
+    $ sudo mysqladmin -u root -p version
+    Enter password: 
+
+    mysql Ver 8.0.19-10 for debian-linux-gnu on x86_64 (Percona Server (GPL), Release '10', Revision 'f446c04')
+    ...
+    Server version      8.0.19-10
+    Protocol version    10
+    Connection          Localhost via UNIX socket
+    UNIX socket         /var/run/mysqld/mysqld.sock
+    Uptime:             4 hours 58 min 10 section
+
+    Threads:    2 Questions:    16 Slow queries: 0 Opens: 139 Flush tables: 3
+    Open tables: 59  Queries per second avg: 0.0000 
+
+You can use `mysqlshow <https://dev.mysql.com/doc/refman/5.7/en/mysqlshow.html>`_ to display database and table information.
+
+.. code-block:: bash
+
+    $ sudo mysqlshow -u root -p 
+    Enter password:
+
+    +---------------------+
+    |      Databases      |
+    +=====================+
+    | information_schema  |
+    +---------------------+
+    | mysql               |
+    +---------------------+
+    | performance_schema  |
+    +---------------------+
+    | sys                 |
+    +---------------------+
+
+
+Loading the Time Zone Tables
+----------------------------
+
+The installation process also creates the time zone tables, but the tables are
+not loaded automatically. The `mysql_tzinfo_to_sql
```

Comment:
what installation process? is this needed if I'm using repo packages or only other binaries

Reply:
Added information on the installation process.